packages/create-docusaurus/templates/shared/npmrc

```diff
@@ -0,0 +1 @@
+legacy-peer-deps=true
```

Comment:
that seems annoying to ship this in our init template 😅

Reply:
I've removed it